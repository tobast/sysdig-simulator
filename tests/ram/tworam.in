6
00001101010001010100001010111
00000000000000000000000000000
00011111111110000000000000000
00011000000010000000011111111
00010000000000000000000000000
00000000000000000000000000000
