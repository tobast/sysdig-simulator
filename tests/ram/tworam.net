INPUT rAddr, doWrite, data1, data2
OUTPUT ram2, ram1
VAR
	rAddr:4, data1:8, data2:16, doWrite, ram1:8, ram2:16
IN
ram1 = RAM 4 8 rAddr doWrite rAddr data1
ram2 = RAM 4 16 rAddr doWrite rAddr data2
