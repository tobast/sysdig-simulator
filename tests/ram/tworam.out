000000000000000000000000
101010000101011110101000
000000000000000000000000
000000000000000011111111
000000001111111100000001
101010000101011110101000
